driver/jvm_tooling.cpp

```diff
@@ -281,6 +282,13 @@ JVM::JVM(std::string_view executable_path, std::string_view seed) {
   options.push_back(JavaVMOption{
       .optionString = const_cast<char *>(fake_pcs_property.c_str())});
```

Comment:
This is not related to this PR, but would it make sense to add a patch to the used libFuzzer version that enables `_with_pc` tracing callbacks. This would enable us to remove the inline assembly code and have a better granularity of the possible values for ARM. I have already created a patch and we can add it to the list of patches for libFuzzer.

Reply:
It would. I am planning to update the libFuzzer version anyway as that allows us to use the new `-1` return value of `LLVMFuzzerTestOneInput` to skip inputs, which in turns means we can get rid of the "replay coverage during the first two runs" hack. When I do that, I can also add the patch and then try to upstream it.